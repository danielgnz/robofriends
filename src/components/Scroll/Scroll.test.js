import React from 'react';
import { shallow } from 'enzyme';
import Scroll from './Scroll';

it('expected to render Scroll component', () => {
	expect(shallow(<Scroll />)).toMatchSnapshot();
})