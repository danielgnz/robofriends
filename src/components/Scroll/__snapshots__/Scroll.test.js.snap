// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expected to render Scroll component 1`] = `
<div
  style={
    Object {
      "border": "1px solid black",
      "height": "500px",
      "overflowY": "scroll",
    }
  }
/>
`;
